apps/glusterfs/app_volume.go

```diff
@@ -246,8 +246,22 @@ func (a *App) VolumeDelete(w http.ResponseWriter, r *http.Request) {
 			return err
 		}
 
-		return nil
+		if !volume.Info.Block {
```

Comment:
We need a document specific to the API and design here in this repo as part of this engineering effort. The overall project is in gluster-kubernetes.